`cargo doc` fails on `sv_parser_pp`

When I try to run `cargo doc` on my project it fails once `sv_parser_pp` is reached with this message:
```
Documenting sv-parser-pp v0.11.2
error[E0275]: overflow evaluating the requirement `(sv_parser_syntaxtree::Symbol, sv_parser_syntaxtree::CycleDelayConstRangeExpression, sv_parser_syntaxtree::Symbol): std::marker::Sync`
    |
    = help: consider increasing the recursion limit by adding a `#![recursion_limit = "256"]` attribute to your crate (`sv_parser_pp`)
```

--- a/sv-parser-pp/src/lib.rs
+++ b/sv-parser-pp/src/lib.rs
@@ -1,4 +1,5 @@
 #![allow(clippy::type_complexity)]
+#![recursion_limit = "256"]
 
 pub mod preprocess;
 pub mod range;

